src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,14 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo:React.FC<{ todo: Todo }> = ({ todo }) => {
+  const userInfo = todo.user && <UserInfo user={todo.user} />;
+  const todoInfoClasses = `TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`;
```

Comment:
Really well done here) It's best practise to keep such values in separate consts